src/domains/citizen-auth-common/03_storage.tf

```
@@ -0,0 +1,60 @@
+resource "azurerm_resource_group" "lollipop_assertions_rg" {
+  name     = format("%s-lollipop-common-rg", local.product)
+  location = var.location
+
+  tags = var.tags
+}
+
+module "lollipop_assertions_storage" {
+  source                     = "git::https://github.com/pagopa/terraform-azurerm-v3//storage_account?ref=v4.3.1"
+  name                       = replace(format("%s-assertions", local.product), "-", "")
```

```suggestion
  name                       = replace(format("%s-assertions", local.product), "-", "")
```
from plan the name will be `iopassertions`, is it expected? or maybe `ioplollipopassertionsst`?

---

it should be short enough. Changed! 👍